File: Kotlin/bee 1933.kt
import java.io.BufferedReader
import java.io.InputStreamReader


fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))

    val (a, b) = br.readLine().split(" ").map {it.toInt()}

    println(maior(a,b))
}

fun maior(a: Int, b: Int): Int {
    if(a > b) {
        return a
    }
    return b
}

File: C++/bee 1933.cpp
#include <iostream>
using namespace std;

int maior(int a, int b) {
    if(a > b) {
        return a;
    }

    return b;
}

int main() {
    int a,b;
    cin >> a >> b;
    cout <<  maior(a,b) << endl;
    return 0;
}
